src/data/general-cs-quiz.ts

```diff
@@ -1,4 +1,59 @@
 const generalCSQuiz = [
+  {
+    Question:
+      "Which of the following is corect about a Tree?",
+    Answer: "Tree is minimally connected.",
+    Distractor1: "Tree can be cyclic.",
+    Distractor2: "Tree must be undirected graph",
+    Distractor3: "Tree must have atleast two nodes",
+    Explanation: 
+      "If we remove any edge from a tree, it creates a two disconnected graph",
+    Link: "https://www.youtube.com/watch?v=09_LlHjoEiY&list=PLWKjhJtqVAbn5emQ3RRG8gEBqkhf_5vxD&t=306s",
+  },
+  {
+    Question:
+      "Which of the following is a constraint for Dijkstra's Single Source Shortest Path Algorithm",
+    Answer: "The graph must only contain non-negative edge weights.",
+    Distractor1: "The graph must be acylic",
+    Distractor2: "The graph must be a balanced binary tree",
+    Distractor3: "The graph must have unique edge weights",
+    Explanation: 
+      "Dijkstra's SSSP algorithm may not work for a graph with negative edge weights. This constraint is to ensure that once a vertex is visited, the optimal distance from source cannot be updated further.",
+    Link: "https://www.youtube.com/watch?v=09_LlHjoEiY&list=PLWKjhJtqVAbn5emQ3RRG8gEBqkhf_5vxD&t=4832s",
+  },
+  {
+    Question:
+      "Which of the following is the most correct statement.",
+    Answer: "Topological sort is possible only for a Directed Acyclic Graph",
+    Distractor1: "Toplogical sort is just an another name for BFS",
+    Distractor2: "All undirected graph have topological ordering",
+    Distractor3: "A Cyclic graph also have topological ordering.",
```

Comment:
```suggestion
    Distractor3: "Acyclic graphs also have topological ordering.",
```

Reply:
Here, I specifically meant a graph with cycles.

Reply:
In this case, cyclic should be lowercase. 
I added that change 👍